src/internal_events/sematext_metrics.rs

```diff
@@ -0,0 +1,28 @@
+use super::InternalEvent;
+use crate::event::metric::{MetricKind, MetricValue};
+use metrics::counter;
+
+#[derive(Debug)]
+pub struct SematextMetricsInvalidMetric {
+    pub value: MetricValue,
+    pub kind: MetricKind,
+}
+
+impl InternalEvent for SematextMetricsInvalidMetric {
+    fn emit_logs(&self) {
+        warn!(
+            message = "invalid metric sent; dropping event.",
+            value = ?self.value,
+            kind = ?self.kind,
+            rate_limit_secs = 30,
+        )
+    }
+
+    fn emit_metrics(&self) {
+        counter!(
+            "invalid_metrics", 1,
+            "component_kind" => "sink",
+            "component_type" => "sematext_metrics",
+        );
+    }
```

Comment:
The `processing_errors` counter is more common for this.
```suggestion
        counter!(
            "processing_errors", 1,
            "component_kind" => "sink",
            "component_type" => "sematext_metrics",
            "error_type" => "invalid_metric",
        );
    }
```